share/jupyter/voila/templates/materialscloud/browser-open.html

```diff
@@ -5,9 +5,9 @@
 {% endblock %}
 
 {% block body %}
-  {% call spinner() %}
+  {% call spinner_mcloud() %}
     <h5>
-      This page should redirect you to voila. If it doesn't, <a href="{{ open_url }}">click here to go to voila</a>.
+        Loading ... 
```

Comment:
Keep the old line here, since this is a precautionary measure, if for some reason the browser cannot move to the next page properly.